### PR TITLE
[MHV-55186] Update all V1 components to V3 web components on SM MHV

### PR DESCRIPTION
**Pull request by vsaleem**
## Summary
- #### V1 Web-Components are retired. This update switches all va-components to utilize the V3 uswds web-components.

> Explanation: 
>The components are currently built so one can flip between v1 and v3. But that bloats the components with code that isn't being used. Once v1 is deprecated we'll commence removing that code.
>
>Also, for the sake of consistency we want to move [va.gov](http://va.gov/) to v3 components within a reasonable time frame. Thus hanging on to v1 versions makes that process take longer. @humancompanion-usds 
>

## Testing done

- _Manual Testing_
> To test, start on the 'Messages' landing page and click through all elemental components, (i.g. Accordion, Alerts, Links, etc.), and use google DOM to check if each component contains "uswds" attribute.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Va.gov - Secure Messaging)*

#### AC2 V3 Components to test:
- Va-Accordion
- Va-Additional-Info
- Va-Alert
- Va-Button
- Va-Modal
- Va-Pagination
- Va-Radio
- Va-Select
- Va-Text-Input
- Va-Textarea

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv/secure-messaging/components/AttachmentsList.jsx [hunk 1]
```diff
@@ -135,7 +136,12 @@ const AttachmentsList = props => {
             status="success"
             onCloseEvent={handleSuccessAlertClose}
           >
-            <p className="vads-u-margin-bottom--0">File attached</p>
+            <h3
+              slot="headline"
+              className="vads-u-margin-bottom--0 vads-u-font-size--base vads-font-family-sans-serif"
+            >
+              File attached
+            </h3>
```
Comment (oleksii-morgun) [inline src/applications/mhv/secure-messaging/components/AttachmentsList.jsx:144]: Was this `h3` suggested by a11y? trying to see the rationale behind replacing `p` with heading tag

**Review by alex-morgun-1 — COMMENTED** [review]
Awesome 😃 Just one minor comment to get clarification on heading

**Review by micahchiang — COMMENTED** [review]
With the [merge of this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/28486) on Friday, you should no longer need to include the `uswds` prop in any of your web component declarations.

**Comment by oleksii-morgun** [comment]
@vsaleem per https://github.com/department-of-veterans-affairs/vets-website/pull/28504#pullrequestreview-1943624224 as we chatted on Friday, at this point we would only need `uswds=false` where we need to keep V1 until we work on fixes

**Comment by vsaleem** [comment]
Closing this pull request as it is no longer needed. These issues are being handled in individual stories. [V3 Component-Issues resulting from upgrade](https://jira.devops.va.gov/browse/MHV-56077)

**Pull request closed without merging**